R instat startup

Patowhiz opened the pull request:
Related to PR #6019 and adds @dannyparsons suggestions in 
[comment](https://github.com/africanmathsinitiative/R-Instat/pull/6019#discussion_r509993908) and  [comment](https://github.com/africanmathsinitiative/R-Instat/pull/6019#discussion_r509995156)

Patowhiz commented:
Changes  "R-packages" to "R packages"
Hides "Instructional videos"
Removes links "More to come"

@dannyparsons @lloyddewit this is ready for review.

New design looks as shown below:

<img width="432" alt="new_deign" src="https://user-images.githubusercontent.com/21179007/96856259-60e87980-1466-11eb-83d7-6ae0f90fced3.PNG">

dannyparsons commented:
Looks good but you seem to have additional changes include here on `dlgSaveAs` and `frmMain`, can they be removed?

Patowhiz commented:
Closing this PR due to unintentional changes made to it.
PR #6038 is the one addressing the issues above.
@dannyparsons I have removed the unintended changes in PR #6038
